### ISSUE TITLE
Visualize stale connection

### ISSUE THREAD

**Issue by johansolve** [Kemmey/Kemmey-TeslaWatch-Public#250]
I'd like to suggest that the UI somehow visualizes when there hasn't been a recent connection to the car, maybe by fading out the UI and make it monochrome (as the original Tesla app), a spinner or some other indication that the shown state is not current.

**Comment by Kemmey** [comment]
@johansolve Hey, sorry for not replying to you directly! I immediately implemented this, and it has since shipped to the store :-D 

Thank you for that suggestion!

**Comment by johansolve** [comment]
Yes I noticed this in the release notes and feel happy every time I see the dimmed UI on the watch now :) 
Thanks!

**Comment by johansolve** [comment]
I was thinking maybe 1 hour is too long? How about 15 minutes for example? I think the Tesla app shows stale UI much sooner.

**Comment by Kemmey** [comment]
No problem - will adjust

**Comment by Kemmey** [comment]
But I'm keeping it at one hour for the widgets. Else they'd always be dimmed
